BestPractices.md

```diff
@@ -158,7 +158,8 @@ According to Apple's [documentation](https://developer.apple.com/library/ios/doc
 
 ```Use a weak reference whenever it is valid for that reference to become nil at some point during its lifetime. Conversely, use an unowned reference when you know that the reference will never be nil once it has been set during initialization.```
 
-##### Example #####
+##### Example
```

Comment:
Make sure that the subsections here the same number of `#` as subsections in the other sections.